eng/common/TestResources/New-TestResources.ps1

```diff
@@ -1001,6 +1013,14 @@ The environment file will be named for the test resources template that it was
 generated for. For ARM templates, it will be test-resources.json.env. For
 Bicep templates, test-resources.bicep.env.
 
+.PARAMETER UseUserCredentials
```

Comment:
IIRC, you need to update the markdown docs. There's a readme on how exactly to do that. Pretty straight forward.